charts/helm-controller-stack/values.yaml

```
@@ -162,17 +162,19 @@ kubeRBACProxy:
   resources: {}
 
 serviceMonitor:
-  # -- Create a rtmp operator ServiceMonitor.
-  enabled: false
-  # -- Labels added to the ServiceMonitor.
-  additionalLabels: {}
-  # -- Annotations added to the ServiceMonitor.
-  annotations: {}
-  # -- Override prometheus operator scrapping interval.
-  interval: ""
-  # -- Override prometheus operator scrapping timeout.
-  scrapeTimeout: ""
-  # -- Relabellings to apply to samples before scraping.
+  # -- If true, a ServiceMonitor CRD is created for a prometheus operator.
```

A ServiceMonitor is not a CRD
```suggestion
  # -- If true, a ServiceMonitor is created for a prometheus operator.
```